test/XmlDataContractSerializerInputFormatterTests.cs

```diff
@@ -1,499 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
I haven't rebased recently from dev, so that's a possibility.  This file was renamed to XmlDataContractSerializerInputFormatterTest (no s at the end) in this branch.  I can rebase onto upstream before I rebase / squash this branch again.